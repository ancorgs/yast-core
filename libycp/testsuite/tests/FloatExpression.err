Parsed:
----------------------------------------------------------------------
(4.20000000000000 + 1.30000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(4.20000000000000 - 1.30000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(4.20000000000000 * 1.30000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(4.20000000000000 / 1.30000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(4.20000000000000 - (1.30000000000000 * 3.00000000000000))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(4.20000000000000 - (1.30000000000000 * 3.00000000000000))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((4.20000000000000 - (1.30000000000000 * 3.00000000000000)) + 3.90000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((4.20000000000000 - (1.30000000000000 * 3.00000000000000)) + 3.90000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((1.30000000000000 - 0.800000000000000) * (0.00000000000000 - 0.500000000000000))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((1.30000000000000 - 0.800000000000000) * (0.00000000000000 + 0.500000000000000))
----------------------------------------------------------------------
