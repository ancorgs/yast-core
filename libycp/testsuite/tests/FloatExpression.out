(5.50000000000000)
(2.90000000000000)
(5.46000000000000)
(3.23076923076923)
(0.300000000000000)
(0.300000000000000)
(4.20000000000000)
(4.20000000000000)
(-0.250000000000000)
(0.250000000000000)
